sys/dev/nvme/nvme_ctrlr.c

```diff
@@ -1273,7 +1273,8 @@ nvme_ctrlr_passthrough_cmd(struct nvme_controller *ctrlr,
 			req = nvme_allocate_request_vaddr(buf->b_data, pt->len, 
 			    nvme_pt_done, pt);
 		} else
```

Comment:
No actual code causes this branch to be taken.  It seems to just be pre-emptive over engineering in the original commit of passthrough support.